src/app/static/src/tests/components/downloadResults/uploadModal.test.ts

```diff
@@ -0,0 +1,157 @@
+    }
+    const createStore = () => {
+        const store = new Vuex.Store({
+            state: emptyState(),
+            modules: {
+                baseline: {
+                    namespaced: true,
+                    state: mockBaselineState(
+                        {
+                            selectedDataset:
+                                {
+                                    id: "1",
+                                    url: "example.com",
+                                    title: "test title",
+                                    resources: mockResources,
+                                    organization: mockOrganization
+                                }
+                        })
+                },
+                adr: {
+                    namespaced: true,
+                    state: mockADRState({uploadFiles: fakeMetadata}),
+                    actions: {
+                        uploadFiles: jest.fn()
+                    }
+                }
+            }
+        });
+        registerTranslations(store);
+        return store;
+    };
+
+    const getWrapper = (data = fakeMetadata) => {
+        return shallowMount(UploadModal,
+            {
+                store: createStore(),
+                propsData: {outputFileMetadata: data}
+            })
+    }
+
+    it(`renders modal as expected`, () => {
+        const wrapper = getWrapper()
+        const store = wrapper.vm.$store
+
+        const dialog = wrapper.find("#dialog")
+        expect(dialog.exists()).toBe(true)
+        const text = dialog.find("h4")
+        expectTranslated(text, "Upload to ADR", "Télécharger vers ADR", store)
+    })
+
+    it(`renders dataset name as expected`, () => {
+        const wrapper = getWrapper()
+        const store = wrapper.vm.$store
+
+        const dataset = wrapper.find("#dialog").find("#dataset-id")
+        expectTranslated(dataset, "Dataset: test title",
+            "Base de données: test title", store)
+    })
+
+    it(`renders instructions text as expected`, () => {
+        const wrapper = getWrapper()
+        const store = wrapper.vm.$store
+
+        const instructions = wrapper.find("#instructions")
+        expectTranslated(instructions,
+            "Please select the new or modified files which should be uploaded",
+            "Veuillez sélectionner les fichiers nouveaux ou modifiés qui doivent être téléchargés",
+            store)
+    })
+
+    it(`renders Output files, inputs, overwritten text and labels as expected`, () => {
+        const wrapper = getWrapper()
+        const store = wrapper.vm.$store
+
+        const overwriteText = wrapper.findAll("small")
+        expect(overwriteText.length).toBe(1)
+        expect(overwriteText.at(0).text()).toBe("This file already exists on ADR " +
+            "and will be overwritten. File was updated 25/01/2021 06:34:12")
+
+        const inputs = wrapper.findAll("input.form-check-input")
+        expect(inputs.length).toBe(2)
```

Comment:
You should also check that the checkboxes are ticked (the default behaviour)

Reply:
Oh yeah, missed this.